### ISSUE TITLE
battery entities missing after removal of batt_entities

### ISSUE THREAD

**Issue by FrancYescO** [custom-components/ble_monitor#456]
Hello i was using the `batt_entities` prop and it was working good, but after the 3.0.0 release (actually i'm on 4.4.0) i'm not able to get again the battery entities of mine RTCGQ02LM..

accordly to the [changelog](https://github.com/custom-components/ble_monitor/releases/tag/3.0.0):

> Removal of the generate battery entities option.
> 
>     The option batt_entities has been removed (it will display a warning if still in your YAML configuration). BLE monitor will, from now on, always generate the battery and/or voltage entities. HA has built in functionality to disable these sensors.
> 

but looks is not my case as i actually i have no battery entities generated (i've remove also the prop from the config yaml)
<img width="331" alt="immagine" src="https://user-images.githubusercontent.com/4574593/127998577-0af0a797-1a6d-458e-b947-612564822b5a.png">

any help to get these entities back?

tried to, remove the ble_monitor block from config, removed integration, restart HA, readd config block, restart HA, but did not work, still have no battery entities generated

**Comment by Ernst79** [comment]
Could you try the following. In config/custom_components/ble_monitor/const.py on line 136, change the following line

```
    'RTCGQ02LM'               : [[], ["button"], ["light", "motion"]],
```

to 

```
    'RTCGQ02LM'               : [["battery"], ["button"], ["light", "motion"]],
```

After that, do a restart. 

It seems that "battery" was accidentally removed. Let me know if that fixes the issue.

**Comment by FrancYescO** [comment]
seems working, actually in unknown status but i assume cuz it still haven't received the update payload for the battery

**Comment by Ernst79** [comment]
Let me know if the battery is updated. It will always start as unknown, till the first advertisement arrives. Can take up to 24 hours.

**Comment by Ernst79** [comment]
Fixed in 4.4.1

**Comment by FrancYescO** [comment]
got battery adv, all okay ;)
